static/src/js/components/GranuleFilters/GranuleFiltersForm.jsx

```diff
@@ -745,6 +771,7 @@ export const GranuleFiltersForm = (props) => {
                         <Col sm={7}>
                           <Form.Control
                             name="equatorCrossingLongitude.min"
+                            aria-label="equator-crossing-longitude-minimum"
```

Comment:
Update these to be sentences instead of snake cased vars

Reply:
We should update all the aria-labels in EDSC for consistency I'll start looking at that.

We may be able to drop the data-test-ids I know at least one playwright test is reliant on them I'll take a look to see if its simple to update it